### PR TITLE
MCA-1270: Add AppSync monitoring

### PR DESCRIPTION
**Pull request by Ben3eeE**

### CODE REVIEW

**Review by henrinie — APPROVED** [review]
Looks good to me. About naming, is there a difference between `nestedAppsync` and then `appsyncApi` or could both be named similarly?

**Comment by Ben3eeE** [comment]
@henrinie All stacks are named `nestedSomething` in the stack folder. That's why it's `nestedAppSync` there.

File: src/stacks/index.ts [hunk 1]
```diff
@@ -7,3 +7,4 @@ export * from './nestedApiGateway';
 export * from './nestedRDS';
 export * from './nestedEKS';
 export * from './monitoring';
+export * from './nestedAppsync';
```
Comment (biphobe) [inline src/stacks/index.ts]: - missing newline :smile: 
- I would keep the same case for all `*AppSync` wordings for consistency, so: `nestedAppSync` here - and there are other places in the PR in which `sync` is lowercase while `App` is uppercase which IMO could get updated

File: src/utils/config.ts [hunk 2]
```diff
@@ -252,6 +254,7 @@ export enum ConfigLocalType {
   RdsInstance = 'rdsInstances',
   EksCluster = 'eksClusters',
   LogGroup = 'logGroups',
+  AppSync = 'appsyncApis',
```
Comment (biphobe) [inline src/utils/config.ts]: I would use uppercase `S` here and in L270 too.

**Review by biphobe — APPROVED** [review]

**Pull request merged**